web/sites/default/settings.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Oomph, Inc. Drupal Scaffold - Drupal 8 Settings
+ *
+ * This file contains the default settings required to have a functioning site
+ * with Lando. You can define your own settings in settings.local.php to get a
+ * working instance of Drupal outside of Lando.
+ *
+ * Note: After a Drupal installation, Drupal _may_ add an installation profile
+ * definition to the end of this file. It is recommended to NOT add this change
+ * to version control. It is safe to delete it if present in this file.
```

Comment:
I believe the install profile in the settings file is deprecated with configuration management. There's no issue if it's omitted AFAIK since the profile is also in the `core.extensions.yml` config.